src/Toast.js

```diff
@@ -89,7 +89,9 @@ const Toast = React.forwardRef(
       onCloseRef.current();
     }, [autohide, show]);
 
-    autohideTimeout.set(autohideFunc, delayRef.current);
+    useEffect(() => {
+      autohideTimeout.set(autohideFunc, delayRef.current);
+    }, [autohide, show, autohideTimeout, autohideFunc]);
```

Comment:
```suggestion
    }, [autohideTimeout, autohideFunc]);
```

Reply:
oops, never mind

Reply:
actually, you probably want a comment noting that `autohide` and `show` are here on purpose.

you may also want to pre-calculate `autohide && show` and use that here

Reply:
Added changes. Thanks!